test/Npgsql.Tests/Replication/TestDecodingReplicationTests.cs

```diff
@@ -339,7 +322,6 @@ public Task Truncate()
 
                     streamingCts.Cancel();
                     await AssertReplicationCancellation(messages);
-                    await rc.DropReplicationSlot(slotName, cancellationToken: CancellationToken.None);
```

Comment:
Does this mean we no longer drop the slot, and it stays around in the database?

BTW that reminds me that during testing I frequently got orphaned slots (e.g. because tests crashed in the middle). I used `SELECT pg_drop_replication_slot(slot_name) FROM pg_replication_slots` to clear all slot in one go - we could put this in the fixture teardown (and possibly even setup).

Reply:
Yeah. Why not.
If you don't abort the test the cleanup code in `SafeReplicationTest` should normally take care of it.

Reply:
After thinking about it I don't think that it's a good idea.
In `SafeReplicationTest` we go a long way to make sure the replication slot is really dropped, even if it is still in use.
The query above will simply fail if a slot is still in use.